config/samples/java_example_loadtest.yaml

```diff
@@ -1,64 +1,75 @@
 apiVersion: e2etest.grpc.io/v1
 kind: LoadTest
 metadata:
-    # Every load test instance must be assigned a unique name on the
-    # cluster. There are ways we can circumvent naming clashes, such
-    # as using namespaces or dynamically assigning names.
-    name: java-example
+  # Every load test instance must be assigned a unique name on the
+  # cluster. There are ways we can circumvent naming clashes, such
+  # as using namespaces or dynamically assigning names.
+  name: java-example
 
-    # As a custom resource, it behaves like a native kubernetes object.
-    # This means that users can perform CRUD operations through the
-    # Kubernetes API or kubectl. In addition, it means that the user
-    # can set any metadata on it.
-    labels:
-        language: java
+  # As a custom resource, it behaves like a native kubernetes object.
+  # This means that users can perform CRUD operations through the
+  # Kubernetes API or kubectl. In addition, it means that the user
+  # can set any metadata on it.
+  labels:
+    language: java
 spec:
-    # The user can specify servers to use when running tests. The
-    # initial version only supports 1 server to limit scope. Servers
-    # is an array for future expansion.
-    #
-    # There are many designs and systems to pursue load balancing,
-    # organizing and monitoring a mesh of servers. Therefore, this
-    # will likely be expanded in the future.
-    servers:
-        - language: java
-          clone:
-            repo: https://github.com/grpc/grpc-java.git
-            gitRef: master
-          build:
-            command: ["gradle"]
-            args: ["-PskipAndroid=true", "-PskipCodegen=true", ":grpc-benchmarks:installDist"]
-          run:
-            command: ["benchmarks/build/install/grpc-benchmarks/bin/benchmark_worker"]
+  # The user can specify servers to use when running tests. The
+  # initial version only supports 1 server to limit scope. Servers
+  # is an array for future expansion.
+  #
+  # There are many designs and systems to pursue load balancing,
+  # organizing and monitoring a mesh of servers. Therefore, this
+  # will likely be expanded in the future.
+  servers:
+    - language: java
+      clone:
+        repo: https://github.com/grpc/grpc-java.git
+        gitRef: master
+      build:
+        command: ["gradle"]
+        args:
+          [
+            "-PskipAndroid=true",
+            "-PskipCodegen=true",
+            ":grpc-benchmarks:installDist",
+          ]
```

Comment:
This is a very interesting choice for prettier. Why did it keep the brackets? It could have been:

```
args:
  - "-PskipAndroid=true"
  - "-PskipCodegen=true"
  - ":grpc-benchmarks:installDist"
```

Reply:
The design decision to use brackets or not was made by the user. Prettier reformatted the file to break up a long line, but kept the original decision. This behavior makes it easier for the user to override the change if they don't agree with it, by moving the items together into one line with only whitespace changes. I think this is a good choice of default behavior by the tool.